libmscore/layout.h

```diff
@@ -13,11 +13,61 @@
 #ifndef __LAYOUT_H__
 #define __LAYOUT_H__
 
+#include "system.h"
+
 namespace Ms {
 
 class Segment;
 class Page;
 
+//---------------------------------------------------------
+//   VerticalStretchData
+//    helper class for spreading staves over a page
+//---------------------------------------------------------
+
+class VerticalGapData {
+      char      c1             { ' '     };
```

Comment:
I see these are set, but not when they are used. Could you clarify their usage with a comment?

Reply:
I'm speaking about `c1`, `c2`, etc. here

Reply:
Sorry, leftovers of debugging code, will be removed.

Reply:
Has been removed.